src/meson.build

```diff
@@ -1,22 +1,25 @@
+python = import('python')
+python_bin = python.find_installation('python3')
+python_dir = join_paths(get_option('prefix'), python_bin.get_install_dir())
+
 pkgdatadir = join_paths(get_option('prefix'), get_option('datadir'), meson.project_name())
-moduledir = join_paths(pkgdatadir, 'portfolio')
+moduledir = join_paths(python_dir, meson.project_name())
 gnome = import('gnome')
 
+conf = configuration_data()
+conf.set('PYTHON', python.find_installation('python3').path())
+conf.set('VERSION', meson.project_version())
+conf.set('localedir', join_paths(get_option('prefix'), get_option('localedir')))
+conf.set('pkgdatadir', pkgdatadir)
+
+
 gnome.compile_resources('portfolio',
   'portfolio.gresource.xml',
   gresource_bundle: true,
   install: true,
   install_dir: pkgdatadir,
 )
 
-python = import('python')
-
-conf = configuration_data()
```

Comment:
it's better not to move there is not strictly necessary, smaller patch :)

Reply:
python must be set before moduledir since moduledir uses it.
Alternatively, I can move moduledir if you prefer, that will make the patch smaller too

Reply:
You can move the import, without moving `conf = configuration_data()`

Reply:
Indeed, the patch is smaller now :-)